cmd/catchpointdump/file.go

```diff
@@ -167,31 +167,36 @@ func isGzipCompressed(catchpointReader *bufio.Reader, catchpointFileSize int64)
 	return prefixBytes[0] == gzipPrefix[0] && prefixBytes[1] == gzipPrefix[1]
 }
 
-func getCatchpointTarReader(catchpointReader *bufio.Reader, catchpointFileSize int64) (*tar.Reader, error) {
+func getCatchpointTarReader(catchpointReader *bufio.Reader, catchpointFileSize int64) (*tar.Reader, bool, error) {
```

Comment:
this bool just represents if the reader is a compressed reader? could use a comment to help point that out